javascript/javascript3/week3/shopping-cart/main.js

```diff
@@ -0,0 +1,118 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    fetch(`https://api.exchangerate-api.com/v4/latest/DKK`)
+      .then((response) => response.json())
+      .then((data) => {
+        let currencyConvert = (data.rates[currency] * this.price).toFixed(2);
+        console.log(
+          `Price of ${this.name} in ${currency} is ${currencyConvert}`
+        );
+      });
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+  }
+
+  addProduct(product) {
+    // Implement functionality here
+    this.products.push(product);
+  }
+
+  removeProduct(product) {
+    // Implement functionality here
+    const removedProduct = this.products.filter(
```

Comment:
this is confusing. You call the variable `removedProduct`, but it actually contains the _opposite_ of that, i.e. everything _except_ the removed product. A better name would be `remainingProducts` or something like that.

Reply:
Done, I renamed it updatedArrayList now